doc/contributing/building_ruby.md

```diff
@@ -44,13 +54,31 @@ about Ruby's build to help out.
 
 ### Running make scripts in parallel
 
-To run make scripts in parallel, pass flag `-j<number of processes>`. For instance,
+To run a specific make script in parallel, pass flag `-j<number of processes>`. For instance,
 to run tests on 8 processes, use:
```

Comment:
This option is particular for some implementations, i.e., GNU make, BSD make.
It should be mentioned clearly.